committeeoversightapp/templatetags/committeeoversight_tags.py

```diff
@@ -0,0 +1,32 @@
+from django import template
+from slugify import slugify
+
+from committeeoversightapp.models import CommitteeOrganization
+
+register = template.Library()
+
+
+@register.filter
+def get_category_url(category):
```

Comment:
A custom filter works perfectly fine for this function (and the rest of the functions in this module), but I often implement operations like this one (i.e. performs some processing on a single object, returns a value instead of e.g. rendered HTML) as methods or `@property` methods on the relevant model class, since it feels more object-oriented and often makes it easier to test.

Not a required change, just an alternate possibility I wanted to point out.

Reply:
I'm guessing maybe that your reasoning here was that these models are OCD models instead of committeeoversight models -- if that's the case, a [proxy model](https://docs.djangoproject.com/en/2.2/topics/db/models/#proxy-models) might be able to give you the access you need to the model class.